tools/admin/wskadmin

```diff
@@ -252,43 +288,94 @@ def getSubjectFromDb(args, props):
     else:
         return (None, res)
 
-def deleteUserCmd(args, props):
+def getIdentitiesFromNamespace(args, props):
     protocol = props[DB_PROTOCOL]
     host     = props[DB_HOST]
     port     = props[DB_PORT]
     username = props[DB_USERNAME]
     password = props[DB_PASSWORD]
     database = props[DB_WHISK_AUTHS]
 
-    if args.subject.strip() == '':
-        print 'Subject must not be empty'
-        return 2
-
-    (rev, res) = getSubjectFromDb(args, props)
-    if rev is None:
-        print 'Failed to delete subject (%s)' % res.read().strip()
-        return 1
-
-    url = '%(protocol)s://%(host)s:%(port)s/%(database)s/%(subject)s?rev=%(rev)s' % {
+    url = '%(protocol)s://%(host)s:%(port)s/%(database)s/_design/subjects/_view/identities?key=["%(ns)s"]' % {
         'protocol': protocol,
         'host'    : host,
         'port'    : port,
+        'username': username,
         'database': database,
-        'subject' : args.subject,
-        'rev'     : rev['_rev']
+        'ns'      : args.namespace
     }
 
     headers = {
         'Content-Type': 'application/json',
     }
 
-    res = request('DELETE', url, headers=headers, auth='%s:%s' % (username, password), verbose=args.verbose)
-    if res.status in [200, 202]:
-        print 'Subject deleted'
-    else:
+    res = request('GET', url, headers=headers, auth='%s:%s' % (username, password), verbose=args.verbose)
+    nslist = None
+    if res.status == 200:
+        doc = json.loads(res.read())
+        nslist = []
+        if 'rows' in doc and len(doc['rows']) > 0:
+            for row in doc['rows']:
+                if 'id' in row:
+                    nslist.append({"subject": row["id"], "uuid": row['value']['uuid'], "key": row['value']['key']})
+    return (nslist, res)
+
+def deleteUserCmd(args, props):
+    protocol = props[DB_PROTOCOL]
+    host     = props[DB_HOST]
+    port     = props[DB_PORT]
+    username = props[DB_USERNAME]
+    password = props[DB_PASSWORD]
+    database = props[DB_WHISK_AUTHS]
+
+    if args.subject.strip() == '':
+        print 'Subject must not be empty'
+        return 2
+
+    if args.namespace and args.namespace.strip() == '':
+        print 'Namespace must not be empty'
+        return 2
```

Comment:
FWIW: Those sanity checks could go to the argument parser directly, making for a more concise reading experience.

For example:

```
def non_empty_string(str):
    if str.strip() == '': raise argparse.ArgumentTypeError("The provided option must not be empty.")
    return str

subcmd.add_argument('subject', help='the subject to get key for', type=non_empty_string)
```

Not blocking the PR for it, but a pattern I generally liked.